action.yml

```diff
@@ -0,0 +1,17 @@
+name: "Sprocket Check"
+author: "St. Jude Cloud Rust Labs"
+description: "Check WDL documents for validity using Sprocket"
+inputs:
+  lint:
+    description: "Whether to lint the WDL document"
+    required: false
+    default: "false"
+outputs:
+  status:
+    description: "The status of the check"
+runs:
+  using: "docker"
+  image: "Dockerfile"
```

Comment:
This causes the action to build from the Dockerfile on each run, which is a bit slow given sprocket's install time.

Reply:
But it also ensures that the latest `sprocket` is run each time, right? The potential downside with a caching solution is missing updates

Reply:
Yes, since I don't pin the spocket version in Dockerfile.